src/main/java/org/folio/domain/ActionBlocks.java

```diff
@@ -134,39 +153,63 @@ public static ActionBlocks empty() {
     return new ActionBlocks(false, false, false);
   }
 
-  private static int getLoanOverdueMinutes(OpenLoan loan, Map<String, Integer> overdueMinutes) {
-    if (loan == null || loan.getLoanId() == null || overdueMinutes.get(loan.getLoanId()) == null) {
-      log.error("Failed to get loan overdue minutes: one of the parameters is null");
+  private static int getLoanOverdueMinutes(OpenLoan openLoan, Map<String, Integer> overdueMinutes) {
+    log.debug("getLoanOverdueMinutes:: parameters openLoan: {}, overdueMinutes: {}",
+      () -> logAsJson(openLoan), () -> overdueMinutes);
+    if (openLoan == null || openLoan.getLoanId() == null ||
+      overdueMinutes.get(openLoan.getLoanId()) == null) {
```

Comment:
all these values are used again below, can we extract them into variables and log properly?

Reply:
We're logging both openLoan and overdueMinutes, no sense in logging parts of them separately. We can't extract variables here (without adding tons of null-checks) because the whole idea of this one-line null-check is that if openLoan is null it'll just exit and we won't cause NPE when checking openLoan.getLoanId() == null.